src/Data/Nat/Properties.agda

```diff
@@ -121,6 +121,10 @@ m ≟ n = map′ (≡ᵇ⇒≡ m n) (≡⇒≡ᵇ m n) (T? (m ≡ᵇ n))
 <ᵇ-reflects-< : ∀ m n → Reflects (m < n) (m <ᵇ n)
 <ᵇ-reflects-< m n = fromEquivalence (<ᵇ⇒< m n) <⇒<ᵇ
 
+n<ᵇ1+n : ∀ n → (n <ᵇ suc n) ≡ true
```

Comment:
I would really like to avoid ending up copying across all proofs for the ordering relations to their boolean counterparts. This doesn't seem to be used in this PR so could it be removed?

Reply:
Apologies... this commit ended up in this part of the overall binomial collection of linked PRs, instead of PR#1928 where it is actually used. 

But you are of course correct that it use should be avoided in favour of `Reflects`... I had been cutting corners to avoid having to grapple  with that. Will remedy!